packages/serve/src/index.ts

```diff
@@ -174,7 +174,8 @@ class ServeCommand {
                         }
                         return accumulator;
                     }, {});
-                    const result = Object.assign({}, compiler.options.devServer);
+                    // TODO fix compatibility with multi compiler mode
+                    const result = Object.assign({}, (compiler.options || {}).devServer);
```

Comment:
Another line with the assign object would be better

Reply:
it is workaround for broken logic, as I described above it will be fixed in other PR